src/doc/man/cargo-info.md

```diff
@@ -0,0 +1,83 @@
+# cargo-info(1)
+
+## NAME
+
+cargo-info --- Display information about a package in the registry. Default registry is crates.io
+
+## SYNOPSIS
+
+`cargo info` [_options_] _crate_@_version_
+
+## DESCRIPTION
+
+This command displays information about a package in the registry. It fetches data from the package's Cargo.toml file
+and presents it in a user-friendly format.
+
+_crate_`@`_version_: Fetch from a registry with a version constraint of "_version_"
+
+If the specified package is a package in the current workspace, it displays the information in the local Cargo.toml.
+
+If no version is specified, it will select the appropriate version based on the Minimum Supported Rust Version (MSRV)
+The following hierarchy is used to determine the MSRV:
+
+* First, the MSRV of the parent directory package is checked, if it exists.
+* If the parent directory package does not specify an MSRV, the minimal MSRV of the workspace is checked.
+* If neither the workspace nor the parent directory package specify an MSRV, the version of the current Rust compiler (
+  rustc --version) is used.
+
```

Comment:
imo its unnecessary to go into this much detail and is likely to go stale as we evolve things.

Reply:
I worry about it too. I just put too many implementations here. Deleted.